depends/packages/qt.mk

```diff
@@ -31,7 +31,7 @@ $(package)_config_opts += -no-iconv
 $(package)_config_opts += -no-gif
 $(package)_config_opts += -no-freetype
 $(package)_config_opts += -no-nis
-$(package)_config_opts += -no-pch
+$(package)_config_opts += -pch
```

Comment:
I am assuming you added precompiled header support because our bitcoin build is unlikely to touch anything related to the graphical components? Is this correct or is there another reason?